build failed

@claude Creating an optimized production build ...
Generated 12 documents in .contentlayer
<w> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Parsing of /home/runner/work/data-blog/data-blog/embuild-analyses/node_modules/@contentlayer/core/dist/generation/generate-dotpkg.js for build dependencies failed at 'import(filePathJoin(generatedPkgPath, 'generated', 'index.mjs'))'.
<w> Build dependencies behind this expression are ignored and might cause incorrect cache invalidation.
<w> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Parsing of /home/runner/work/data-blog/data-blog/embuild-analyses/node_modules/@contentlayer/core/dist/generation/generate-dotpkg.js for build dependencies failed at 'import(filePathJoin(generatedPkgPath, 'generated', 'index.mjs'))'.
<w> Build dependencies behind this expression are ignored and might cause incorrect cache invalidation.
<w> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Parsing of /home/runner/work/data-blog/data-blog/embuild-analyses/node_modules/@contentlayer/core/dist/generation/generate-dotpkg.js for build dependencies failed at 'import(filePathJoin(generatedPkgPath, 'generated', 'index.mjs'))'.
<w> Build dependencies behind this expression are ignored and might cause incorrect cache invalidation.
 ✓ Compiled successfully
   Linting and checking validity of types ...
Failed to compile.

./src/components/analyses/vergunningen-goedkeuringen/VergunningenDashboard.tsx:70:17
Type error: Cannot find name 'data'.

  68 |         <AnalysisSection
  69 |           title="Renovatie (Gebouwen)"
> 70 |           data={data as DataRow[]}
     |                 ^
  71 |           municipalities={municipalities}
  72 |           metric="ren"
  73 |           label="Aantal"
Error: Process completed with exit code 1.

--- a/embuild-analyses/src/components/analyses/vergunningen-goedkeuringen/VergunningenDashboard.tsx
+++ b/embuild-analyses/src/components/analyses/vergunningen-goedkeuringen/VergunningenDashboard.tsx
@@ -34,13 +34,19 @@ export function VergunningenDashboard() {
         setLoading(true)
         setError(null)
 
+        // Use environment variable for basePath (empty in dev, /data-blog in production)
+        const basePath = process.env.NEXT_PUBLIC_BASE_PATH || ""
+
         const [dataResponse, municipalitiesResponse] = await Promise.all([
-          fetch("/data-blog/data/vergunningen-goedkeuringen/data_quarterly.json"),
-          fetch("/data-blog/data/vergunningen-goedkeuringen/municipalities.json"),
+          fetch(`${basePath}/data/vergunningen-goedkeuringen/data_quarterly.json`),
+          fetch(`${basePath}/data/vergunningen-goedkeuringen/municipalities.json`),
         ])
 
-        if (!dataResponse.ok || !municipalitiesResponse.ok) {
-          throw new Error("Failed to load data files")
+        if (!dataResponse.ok) {
+          throw new Error(`Kon data_quarterly.json niet laden (HTTP ${dataResponse.status})`)
+        }
+        if (!municipalitiesResponse.ok) {
+          throw new Error(`Kon municipalities.json niet laden (HTTP ${municipalitiesResponse.status})`)
         }
 
         const [dataJson, municipalitiesJson] = await Promise.all([
@@ -48,10 +54,18 @@ export function VergunningenDashboard() {
           municipalitiesResponse.json(),
         ])
 
+        // Validate data is an array
+        if (!Array.isArray(dataJson)) {
+          throw new Error("Ongeldige data formaat voor data_quarterly.json")
+        }
+        if (!Array.isArray(municipalitiesJson)) {
+          throw new Error("Ongeldige data formaat voor municipalities.json")
+        }
+
         setData(dataJson)
         setMunicipalities(municipalitiesJson)
       } catch (err) {
-        setError(err instanceof Error ? err.message : "Failed to load data")
+        setError(err instanceof Error ? err.message : "Kon databestanden niet laden")
       } finally {
         setLoading(false)
       }

